guides/doc-Administering_Red_Hat_Satellite/topics/Maintenance.adoc

```diff
@@ -160,7 +160,7 @@ If you want to check for updates using `yum`, enter the command to install and u
 # {foreman-maintain} packages lock
 ----
 Updating packages individually can lead to package inconsistencies in {Project} or {SmartProxy}.
-For more information about updating packages in {Project}, see https://access.redhat.com/documentation/en-us/red_hat_satellite/{ProductVersion}/html/upgrading_and_updating_red_hat_satellite/updating_satellite_server_capsule_server_and_content_hosts#updating_satellite_server_to_next_minor_version[Updating {ProjectServer}].
+For more information about updating packages in {Project}, see https://access.redhat.com/documentation/en-us/red_hat_satellite/{AccessRedHatComVersion}/html/upgrading_and_updating_red_hat_satellite/updating_satellite_server_capsule_server_and_content_hosts#updating_satellite_server_to_next_minor_version[Updating {ProjectServer}].
```

Comment:
We now have this guide upstream so we should be using the correct attribute for it.

Reply:
:UpgradingDocURL: {BaseURL}Upgrading_and_Updating/index-foreman-el.html# etc in attributes-base

Reply:
Fixed. Can you please re-review this?